polaris/ocean/tasks/merry_go_round/init.py

```diff
@@ -0,0 +1,157 @@
+        """
+        config = self.config
+        logger = self.logger
+
+        section = config['merry_go_round']
+        temperature_left = section.getfloat('temperature_left')
+        temperature_right = section.getfloat('temperature_right')
+        tracer2_background = section.getfloat('tracer2_background')
+        tracer3_background = section.getfloat('tracer3_background')
+        salinity_background = section.getfloat('salinity_background')
+
+        vert_coord = config.get('vertical_grid', 'coord_type')
+        bottom_depth = config.getfloat('vertical_grid', 'bottom_depth')
+
+        if not (vert_coord == 'z-level' or vert_coord == 'sigma'):
+            raise ValueError('Vertical coordinate {vert_coord} not supported')
+
+        lx, ly = (500, 5)
+        nx, ny = compute_planar_hex_nx_ny(lx, ly, self.resolution)
+        nz = int(50 / (self.resolution / 5))
+
+        ds_mesh = make_planar_hex_mesh(nx=nx, ny=ny, dc=1e3 * self.resolution,
+                                       nonperiodic_x=False,
+                                       nonperiodic_y=True)
+        write_netcdf(ds_mesh, 'base_mesh.nc')
+
+        ds_mesh = cull(ds_mesh, logger=logger)
+        ds_mesh = convert(ds_mesh, graphInfoFileName='culled_graph.info',
+                          logger=logger)
+        write_netcdf(ds_mesh, 'culled_mesh.nc')
+
+        ds = ds_mesh.copy()
+        x_cell_adjusted = ds.xCell - ds.xEdge.min()
+        x_edge_adjusted = ds.xEdge - ds.xEdge.min()
+
+        ds['ssh'] = xr.ones_like(ds.xCell)
+        ds['bottomDepth'] = bottom_depth * xr.ones_like(ds.xCell)
+
+        config.set('vertical_grid', 'vert_levels', str(nz))
+        init_vertical_coord(config, ds)
+
+        x_min = x_cell_adjusted.min()
+        x_max = x_cell_adjusted.max()
+        x_mid = 0.5 * (x_min + x_max)
+        z_mid = ds.zMid.squeeze("Time")
+
+        # Only the top layer moves in this test case
+        ds["vertCoordMovementWeights"] = xr.ones_like(ds.refZMid)
+
+        if vert_coord == 'z-level':
+            ds["vertCoordMovementWeights"][:] = 0.0
+            ds["vertCoordMovementWeights"][1] = 1.0
+
+        # ...
+        x_cell_2D, _ = xr.broadcast(x_cell_adjusted, ds.refBottomDepth)
+        x_edge_2D, _ = xr.broadcast(x_edge_adjusted, ds.refBottomDepth)
+        angle_edge_2D, _ = xr.broadcast(ds.angleEdge, ds.refBottomDepth)
+
+        # Initialize temperature
+        temperature = temperature_right * xr.ones_like(x_cell_2D)
+        temperature = xr.where(
+            x_cell_2D < x_mid, temperature_left, temperature_right
+        )
+        temperature = temperature.expand_dims(dim='Time', axis=0)
+        ds['temperature'] = temperature
+
+        # Initialize temperature
+        ds['salinity'] = salinity_background * xr.ones_like(temperature)
+
+        # Initialize normalVelocity
+        z_mid_edge = z_mid.isel(nCells=ds.cellsOnEdge - 1).mean("TWO")
+
+        x_one_quarters = 0.75 * x_min + 0.25 * x_max
```

Comment:
```suggestion
        x_one_quarters = x_min + 0.25 * (x_max - x_min)
```
This is more intuitive to me

Reply:
Addressed in [80ad6f1](https://github.com/E3SM-Project/polaris/pull/284/commits/80ad6f149cd4755b7ee7248ab78ce2825ade6557).

Reply:
Probably should have done this through the commit suggestion functionality here.  My bad.